modules/serverless-logic-workflow-base-builder-dockerfile.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: CONCEPT
+[id="serverless-logic-workflow-base-builder-dockerfile_{context}"]
+= Customize the base builder Dockerfile
```

Comment:
This is more of a reference than a concept, and the title sort of sounds like it's a procedure. Suggest changing the content type and tweaking the title:
```suggestion
:_content-type: REFERENCE
[id="serverless-logic-workflow-base-builder-dockerfile_{context}"]
= Customization for the base builder Dockerfile
```